### PR TITLE
Fix line spacing

### PR DESCRIPTION
**Pull request by tjachu**
## Description

- Home page line spacing changes (landing page & contact info) 
- "Who is this for?" card components, fix width and center icon

## Updates

[Please enter a brief summary of updates to this PR during code review - if any]

## Checklist

- [ ] Ran `npm run lint:fix` to format code
- [ ] Requested at least one reviewer
- [ ] Connected PR to Trello ticket (steps below)
- [ ] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

### CODE REVIEW

File: package-lock.json
Comment (ElisaHu1) [inline package-lock.json:1]: you do not need to add the change of package-lock.json

File: src/components/LandingPage/LandingPage.css [hunk 1]
```diff
@@ -26,11 +27,12 @@
 #lab-description {
   font-weight: 400;
   font-size: 30px;
-  line-height: 24px;
-  letter-spacing: 0.04em;
+  line-height: 20px;
```
Comment (ryanyae) [inline src/components/LandingPage/LandingPage.css:30]: I think we can just take this line of code out and the other "line-height:1.2" to completely fix this issue, as I am still seeing a little bit of over lap still. 
image

File: src/components/About/About.css [hunk 2]
```diff
@@ -214,7 +215,7 @@ h3 {
 }
 
 .lab-address {
-  line-height: 16px;
+  line-height: 22px;
```
Comment (ryanyae) [inline src/components/About/About.css:218]: Taking this piece of code out aswell will take the overlapping of text. (Image for the change you have as of now) 
image

**Review by ryanyae — CHANGES_REQUESTED** [review]
I think having the line-height values for all of the text may be the best option. Besides, the two comments that I had left, great work on the ticket.

**Review by ElisaHu1 — APPROVED** [review]
looks great!

**Review by ryanyae — APPROVED** [review]
Looks good to me, thank you Allison!

**Pull request merged**